sdk/deviceupdate/azure-iot-deviceupdate/azure/iot/deviceupdate/_device_update_client.py

```diff
@@ -6,84 +6,89 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from typing import TYPE_CHECKING
+from copy import deepcopy
+from typing import Any, Optional, TYPE_CHECKING
 
 from azure.core import PipelineClient
+from azure.core.rest import HttpRequest, HttpResponse
 from msrest import Deserializer, Serializer
 
+from ._configuration import DeviceUpdateClientConfiguration
+from .operations import DeviceManagementOperations, DeviceUpdateOperations
+
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
-    from typing import Any
+    from typing import Dict
 
     from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
-
-from ._configuration import DeviceUpdateClientConfiguration
-from .operations import UpdatesOperations
-from .operations import DevicesOperations
-from .operations import DeploymentsOperations
-from . import models
 
-
-class DeviceUpdateClient(object):
+class DeviceUpdateClient:
     """Device Update for IoT Hub is an Azure service that enables customers to publish update for their IoT devices to the cloud, and then deploy that update to their devices (approve updates to groups of devices managed and provisioned in IoT Hub). It leverages the proven security and reliability of the Windows Update platform, optimized for IoT devices. It works globally and knows when and how to update devices, enabling customers to focus on their business goals and let Device Update for IoT Hub handle the updates.
 
-    :ivar updates: UpdatesOperations operations
-    :vartype updates: azure.iot.deviceupdate.operations.UpdatesOperations
-    :ivar devices: DevicesOperations operations
-    :vartype devices: azure.iot.deviceupdate.operations.DevicesOperations
-    :ivar deployments: DeploymentsOperations operations
-    :vartype deployments: azure.iot.deviceupdate.operations.DeploymentsOperations
-    :param credential: Credential needed for the client to connect to Azure.
-    :type credential: ~azure.core.credentials.TokenCredential
-    :param account_endpoint: Account endpoint.
-    :type account_endpoint: str
+    :ivar device_update: DeviceUpdateOperations operations
```

Comment:
just want to make sure these name changes are expected, since DeviceUpdateClient().device_update etc seems kind of repetitive

Reply:
It is expected since service team change swagger name with agreement of Mike : https://github.com/Azure/azure-rest-api-specs/pull/17329

Reply:
I agree with @iscai-msft - we will probably need to add directives to opt-out of this operation group name change in Python. Not necessary for this preview but we will want to fix this for the next preview.